mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -103,20 +103,23 @@ class MqttSessionSpec
       val connect = Connect("some-client-id", ConnectFlags.None)
       val carry = "some-carry"
 
-      val result =
+      val (client, result) =
         Source
-          .single(Command(connect, carry))
+          .queue[Command[String]](1, OverflowStrategy.fail)
```

Comment:
This and the next test are changed so that we don't complete the stream prematurely. Now that we process the connection being lost this and the next test would fail without these changes (and rightly so).